### PR TITLE
Added BuyMeACoffee in Contributors domain

### LINKED ISSUE #13163: Add BuyMeACoffee.com to contributors domains
See also:
https://github.com/mozilla/addons-server/issues/11680

Buy me a coffee is a nice alternative as it allows one time donation without account. Also money are transferred directly to your account.

Here comparison with Patreon:
https://www.buymeacoffee.com/patreon-alternative

Ever since I've add it to my extensions, people use it more often than PayPal. I guess it feels better to actually buy something for someone (even if it's virtual) rather than send them money directly.
My link:
https://www.buymeacoffee.com/fastaddons

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes #13163 

I have added BuyMeACoffee in the validContribiutors domain.

* [x] This PR relates to an existing open issue and there are no existing
      PRs open for the same issue.
* [x] Add `Fixes #ISSUENUM` at the top of your PR.
* [x] Add a description of the changes introduced in this PR.
* [x] The change has been successfully run locally.

### CODE REVIEW

**Comment by abhinavraj23** [comment]
@caitmuenster I have added BuyMeACoffee to the valid contributors domain, please review and let me know if I need to do something else here.

**Comment by bobsilverberg** [comment]
@jvillalobos I assume, as you added the labels to #13163, that we are okay with this change, but as this is my first time reviewing a change to this file I figured I should double check with you. Are we fine adding this to the whitelisted list of sites?

File: src/olympia/constants/base.py [hunk 1]
```diff
@@ -204,6 +204,7 @@
     'opencollective.com',
     'patreon.com',
     'paypal.com',
+    'buymeacoffee.com',
```
Comment (bobsilverberg) [inline src/olympia/constants/base.py]: It looks like this list of domains is currently in alphabetical order. Could you please move this to the top of the list?

Comment (abhinavraj23) [reply src/olympia/constants/base.py]: Sure @bobsilverberg will make that change ASAP!

**Comment by abhinavraj23** [comment]
@bobsilverberg I have made the change, please review it and if its okay then please merge it

File: src/olympia/constants/base.py (hunk 1 above)
Comment (bobsilverberg) [reply src/olympia/constants/base.py]: Thanks!

**Comment by abhinavraj23** [comment]
@bobsilverberg this is my first contribution to Mozilla, I want to keep contributing, this was the good-first-bug assigned by @caitmuenster, can you please help me with next steps and assigning me a new issue or a mentor, thanks!

**Comment by jvillalobos** [comment]
@bobsilverberg Yeah, I reviewed the site and I'm okay with adding it to the list.

**Comment by bobsilverberg** [comment]
> @bobsilverberg this is my first contribution to Mozilla, I want to keep contributing, this was the good-first-bug assigned by @caitmuenster, can you please help me with next steps and assigning me a new issue or a mentor, thanks!

Congratulations @abhinavraj23, well done! I think your best bet is to look at the other bugs tagged with `contrib: welcome`, which can be found [here](https://github.com/mozilla/addons-server/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22contrib%3A+welcome%22).

**Comment by caitmuenster** [comment]
Yay! Congrats @abhinavraj23. 🎉 Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition). Would you also like to create an account on [mozillians.org](https://mozillians.org/e)? I'd be happy to vouch for you. 

> I think your best bet is to look at the other bugs tagged with contrib: welcome, which can be found here.

Yep! You're also welcome to work on [other good-first-bugs](https://addons-pm.herokuapp.com/contrib/good-first-bugs/?dir=desc&sort=updatedAt). :) We assign mentors for each good-first-bug in the issue's comments. If you want to work on a contrib:welcome issue and there's no mentor assigned, please add a comment saying you'd like to work on it and we'll assign a mentor. :) 

Welcome onboard!

**Comment by abhinavraj23** [comment]
@caitmuenster Yes I would love to create an account on [mozilla.org](https://mozillians.org). Please do vouch for me, and I am starting to look for other bugs to solve, will try to contribute as much as I can.

**Pull request merged**